src/components/AccordionItem.tsx

```diff
@@ -0,0 +1,24 @@
+import { useState } from "react";
+
+const AccordionItem = ({ question, answer }: { question: string; answer: string }) => {
+  const [isOpen, setIsOpen] = useState(false);
+
+  const handleClick = () => {
+    console.log("Accordion clicked");
+    setIsOpen(prevState => !prevState);
+  };
+
+  return (
+    <div className="py-6 border-b border-bg-primary/30 cursor-pointer" onClick={handleClick}>
+      <div className="flex items-center justify-between">
+        <span className="flex-1 text-lg font-bold pr-3 block max-w-sm md:w-full">{question}</span>
```

Comment:
The length of the question is too short and doesn't fit design.

<img width="898" alt="Screenshot 2024-08-11 at 1 32 39 PM" src="https://github.com/user-attachments/assets/0f82105d-b62c-4bec-9575-a3ee31a5f308">

Reply:
Fixed